### PR TITLE
Fix aria2

### LINKED ISSUE #5648: [aria2] broken, needs libssh2.so
## Description
After a fresh install of chromebrew, installed aria2.
Ran:
```
$ aria2c
aria2c: error while loading shared libraries: libssh2.so.1: cannot open shared object file: No such file or directory
```

Potential solutions:
1. Compile aria2 without libssh2
2. Put libssh2 in aria2's dependencies.

### PR DESCRIPTION
**Pull request by saltedcoffii**
Fixes #5648

### CODE REVIEW

File: packages/aria2.rb [hunk 1]
```diff
@@ -4,34 +4,21 @@ class Aria2 < Package
...
-     x86_64: 'https://downloads.sourceforge.net/project/chromebrew/x86_64/aria2-1.35.0-2-chromeos-x86_64.tar.xz'
-  })
-  binary_sha256({
-    aarch64: 'b8082b1a4f9b8941f2891564046415abf544f113a452650d139c64253f4378f1',
-     armv7l: 'b8082b1a4f9b8941f2891564046415abf544f113a452650d139c64253f4378f1',
-       i686: '0e3b881cd0505eed0edb1ab898e5a272a3d9d950f9e9c9fd5ebc4d1d9a8c66d9',
-     x86_64: '20cb509c94ac4776c39172fbf8b13c80088249ab6deffce4c4eb6d40081bbd71'
-  })
-
   depends_on 'libgcrypt'
```
Comment (uberhacker) [inline packages/aria2.rb:13]: Don't we also need `depends_on 'libssh2'`?

File: packages/aria2.rb [hunk 2]
```diff
@@ -4,34 +4,21 @@ class Aria2 < Package
...
   def self.build
-    system "env CFLAGS='-fuse-ld=gold -flto' CXXFLAGS='-fuse-ld=gold -flto' \
-     LDFLAGS='-flto' \
+    system "env #{CREW_ENV_OPTIONS}
      ./configure #{CREW_OPTIONS} \
       --without-libnettle \
       --with-libgcrypt \
-      --disable-dependency-tracking"
+      --disable-dependency-tracking \
+      --without-libssh2"
```
Comment (uberhacker) [inline packages/aria2.rb:21]: What's wrong with including this?

File: packages/aria2.rb (hunk 1 above)
Comment (saltedcoffii) [reply packages/aria2.rb:13]: No, because of `--without-libssh2`

File: packages/aria2.rb (hunk 2 above)
Comment (saltedcoffii) [reply packages/aria2.rb:21]: Adds another dependency we don't need.

Comment (uberhacker) [reply packages/aria2.rb:21]: You mean like using ssh 1 instead of 2 which is less secure?

Comment (saltedcoffii) [reply packages/aria2.rb:21]: It just doesn't link against ssh

**Pull request merged**